backend/src/akvo/lumen/endpoint/split_column.clj

```diff
@@ -18,14 +18,15 @@
   (fn [{{:keys [dataset-id]} :path-params
         query-params :query-params
         tenant :tenant}]
-    (let [query           (json/parse-string (get query-params "query") keyword)
-          tenant-conn     (p/connection tenant-manager tenant)
-          dataset-version (db.transformation/latest-dataset-version-by-dataset-id tenant-conn {:dataset-id dataset-id})
-          sql-query       {:table-name  (:table-name dataset-version)
-                           :column-name (:columnName query)
-                           :limit       (str (:limit query "200"))}
-          values          (map (comp str (keyword (:columnName query)))
-                               (db.transformation/select-random-column-data tenant-conn sql-query))
+    (let [query            (json/parse-string (get query-params "query") keyword)
+          namespace        (:namespace query "main")
```

Comment:
we need to update client-side to send `namespace` value

Reply:
https://github.com/akvo/akvo-lumen/pull/2874

Reply:
https://github.com/akvo/akvo-lumen/pull/2874/commits/da8a88ad14da9ef07fa92eb2fab5dd0d971bfdbf